packages/cli-hooks/biome.json

```diff
@@ -1,4 +1,4 @@
 {
-  "$schema": "https://biomejs.dev/schemas/1.8.3/schema.json",
+  "$schema": "https://biomejs.dev/schemas/2.0.5/schema.json",
```

Comment:
note: Each package extends the base `biome.json`. This is why we need to have 1 PR that upgrades all packages to Biome v2 - because the schema is not compatible with Biome 1.x.